pkg/kubeapiserver/options/authentication_test.go

```diff
@@ -1238,7 +1238,7 @@ func TestLoadAuthenticationConfig(t *testing.T) {
 			file: func() string {
 				return writeTempFile(t,
 					`{
-						"apiVersion":"apiserver.config.k8s.io/v1alpha1",
+						"apiVersion":"apiserver.config.k8s.io/v1",
```

Comment:
for the tests that aren't a version-specific test case, updating in place to v1 is fine

for the ones that had a version in the test case name, prefer cloning the v1beta1 cases to new v1 cases and leave the v1alpha1 / v1beta1 cases as-is... when we delete alpha types, we'll drop the alpha test cases

Reply:
done!